src/players/base-player.ts

```diff
@@ -115,9 +116,8 @@ export class BasePlayer implements Player {
     const currentTime = this.getCurrentTime();
 
     // Skip time loaded late.
-    if (isInInterval(startTime, endTime, currentTime, offset)) {
+    if (isInInterval(startTime, startTime + 0.25, currentTime)) {
```

Comment:
0.25 :?

Reply:
This is to stop auto skipping if the provider remembers the previous time you were at so if the skip time starts at 0 and ends at 60 and you stopped the video at 30, if you reload the page it will auto skip to 60

Reply:
This may or may not be desireable, if it is, we can revert the change